frontend/src/scenes/userLogic.tsx

```diff
@@ -111,11 +119,12 @@ export const userLogic = kea<userLogicType<UserType, EventProperty, UserUpdateTy
 
                         posthog.register({
                             posthog_version: user.posthog_version,
-                            has_slack_webhook: !!user.team.slack_incoming_webhook,
+                            has_slack_webhook: !!user.team?.slack_incoming_webhook,
```

Comment:
Note: this will cause conflicts with your other PR

Reply:
Well well wouldn't it be convenient to just do both things in one huge PR which would avoid that conflict… no, actually not, this is how I end up with overly large PRs.